qiskit/result/result.py

```diff
@@ -181,6 +182,87 @@ def get_counts(self, experiment=None):
         except KeyError:
             raise QiskitError('No counts for experiment "{0}"'.format(experiment))
 
+    def get_assertion(self, experiment=None):
+        """
+        Calculate and collect results of statistical tests for an experiment.
+
+        Args:
+            experiment (QuantumCircuit): a breakpoint
+
+        Returns:
+            tuple: tuple containing:
+
+                chisq(float): the chi-squared value
+                pval(float): the p-value
+                passed(Boolean): if assertion passed
+        """
+        exp = experiment
```

Comment:
it seems unnecessary

Reply:
[resolved by this commit](https://github.com/Qiskit/qiskit-terra/pull/2867/commits/1a040382dfc8495c056438c110bd4a68b7d85f2c)